app/components/ConfirmTrade.tsx

```diff
@@ -65,27 +77,92 @@ export const ConfirmTrade = ({
     ? formatEtherWithFixedDecimals(tokenAmountOut)
     : '';
 
+  const waitingTxToast = (txHash: string) =>
+    toast({
+      children: (
+        <div className="flex items-center space-x-4">
+          <Spinner className="h-5 w-5 shrink-0 animate-spin" />
+          <div>
+            Wating for transaction success{' '}
+            <a
+              href={`https://gnosisscan.io/tx/${txHash}`}
```

Comment:
we have a GNOSIS_SCAN_URL constant, let's use that, we could probably have a function to create a `blockExplorerTxLink`.

And maybe GNOSIS_SCAN_URL -> BLOCK_EXPLORER_URL would be better, cause we are not tied to gnosis scan, we can even choose to use other like blockscout.com at some point.

Reply:
Created a task for it https://linear.app/swaprhq/issue/PGIO-119/create-function-to-return-explorer-link